repository_service_tuf_worker/repository.py

```diff
@@ -1519,3 +1520,52 @@ def sign_metadata(
                 "bootstrap": "Bootstrap Finished",
             },
         )
+
+    def delete_sign_metadata(
+        self,
+        payload: Dict[str, Any],
+        update_state: Optional[
+            Task.update_state
+        ] = None,  # It is required (see: app.py)
+    ) -> Dict[str, Any]:
+        role: str = payload.get("role")
+        if role is None:
+            msg = "Deletion of metadata pending signatures failed"
+            return self._task_result(
+                TaskName.DELETE_SIGN_METADATA,
+                False,
+                {"message": msg, "error": "No role provided for deletion."},
+            )
+
+        signing_status = self._settings.get_fresh(f"{role.upper()}_SIGNING")
+        if signing_status is None:
+            return self._task_result(
+                TaskName.DELETE_SIGN_METADATA,
+                False,
+                {
+                    "message": f"Deletion of {role} metadata failed.",
+                    "error": f"The {role} role is not in a signing process.",
+                },
+            )
+
+        self.write_repository_settings(f"{role.upper()}_SIGNING", None)
+        m = f"Deletion of {role} metadata succesfull, signing process stopped"
```

Comment:
To keep consistent as line 1533

```suggestion
        msg = f"Deletion of {role} metadata successful, signing process stopped"
```